app/views/user_mailer/email_confirmation.text.haml

```diff
@@ -0,0 +1,3 @@
+Hi user,
+This is an email from Epic Logger.
+Follow the url to confirm youre account: link.
```

Comment:
there is no link provided here. Go ahead and add a link_to so people can copy that link.

Also, use the username, not a generic `Hi user`.